README.md

```diff
@@ -24,29 +24,15 @@ Following the same concept of [Drupal 8 Experimental modules](https://www.drupal
 some submodules are marked as "Experimental". They are provided for evaluation and testing purposes but are still 
 considered to be in development. Experimental modules are included in the "Apigee (Experimental)" package
  on the "Extend" page of a Drupal site (/admin/modules).
+ 
+## Requirements
 
-## Installing
-
-**Note**: The Apigee Edge module requires Drupal 8.6.x or higher and PHP 7.1 or higher.
-
-The Apigee Edge module may require Drupal core or contributed module
-patches to be able to work properly. These patches can be applied
-automatically when Apigee Edge module gets installed but for that your
-Drupal installation must fulfill the following requirements:
-
-1. [cweagans/composer-patches](https://packagist.org/packages/cweagans/composer-patches) >= 1.6.5 has to be installed.
-2. ["Allowing patches to be applied from dependencies
-"](https://github.com/cweagans/composer-patches/tree/1.6.5#allowing-patches-to-be-applied-from-dependencies)
-has to be enabled in Drupal's composer.json.
-3. Proper [patch level](https://github.com/cweagans/composer-patches/pull/101#issue-104810467)
-for drupal/core has to be set in Drupal's composer.json.
-
-You can find the currently required patches in the Apigee Edge module's [composer.json](https://github.com/apigee/apigee-edge-drupal/blob/8.x-1.x/composer.json)
-and in the Apigee Edge PHP API Client's [composer.json](https://github.com/apigee/apigee-client-php/blob/2.x/composer.json).
+* The Apigee Edge module requires **Drupal 8.7.x** or higher and PHP 7.1 or higher.
+* Drupal 8's minimum requirement is phpdocumentor/reflection-docblock:2.0.4 but at least 3.0 is required by this module. If you get the error  "Your requirements could not be resolved to an installable set of packages" it may be because you are running reflection-docblock version 2. You can update `phpdocumentor/reflection-docblock` with the following command: `composer update phpdocumentor/reflection-docblock --with-dependencies`.
+* **Check [composer.json](https://github.com/apigee/apigee-edge-drupal/blob/8.x-1.x/composer.json) for any required patches.** Patches prefixed with "(For testing)" are only required for running tests. Those are not necessary for using this module. Patches can be applied with the [cweagans/composer-patches](https://packagist.org/packages/cweagans/composer-patches) plugin automatically or manually. See [Applying Patches](#applying-patches) section below.
```

Comment:
I think this can be removed.

Reply:
See my reply to your comment on the "Applying Patches" section.